docs/user/tutorial.rst

```diff
@@ -1304,10 +1304,201 @@ Inspecting the application now returns:
         ⇒ /images/{name} - Item:
            └── GET - on_get
 
-.. Query Strings
-.. -------------
+Query Strings
+-------------
+Now that we are able to get the images from the service, we need a way to get a list available images. We have already set up this route. Before testing this route let's change its output format back to JSON to have a more terminal-friendly output. The top of file ``images.py`` should look like this:
```

Comment:
Could you shorten Rst text lines to a reasonable length, preferably around 80 characters?

Reply:
Done